modules/gapi/test/gapi_fluid_test_kernels.cpp

```diff
@@ -448,6 +449,43 @@ GAPI_FLUID_KERNEL(FSum2MatsAndScalar, TSum2MatsAndScalar, false)
     }
 };
 
+GAPI_FLUID_KERNEL(FSumArrayToMat, TSumArrayToMat, false)
+{
+    static const int Window = 1;
+    static const int LPI    = 2;
+
+    static void run(const std::vector<cv::Mat>    &arr,
+                          cv::gapi::fluid::Buffer &out)
+    {
+        std::vector<const uint8_t*> vec(arr.size());
+        for(unsigned int j = 0; j < arr.size(); ++j)
+        {
+            vec[j] = arr[j].ptr<uint8_t>();
+        }
+
+        static int row = 0;
```

Comment:
This `static` looks a little dirty

Reply:
It shouldn't be here. Your `row` is `out.y()`

Reply:
Changed the kernel